trailblazer/apps/slurm/api.py

```diff
@@ -1,10 +1,50 @@
+import subprocess
+from pathlib import Path
 from typing import Callable, Dict, List, Optional
 
 from trailblazer.apps.slurm.models import SqueueResult
 from trailblazer.apps.slurm.utils import formatters
 from trailblazer.constants import FileFormat, SlurmJobStatus, TrailblazerStatus
 from trailblazer.exc import EmptySqueueError
-from trailblazer.io.controller import ReadStream
+from trailblazer.io.controller import ReadFile, ReadStream
+
+
+def _get_squeue_jobs_flag_input(slurm_job_id_file_content: Dict[str, List[str]]) -> str:
+    """Return a string of comma separated SLURM job ids to be used as input for squeue jobs flag."""
+    job_ids: List[str] = []
+    for slurm_job_ids in slurm_job_id_file_content.values():
+        [job_ids.append(str(job_id)) for job_id in slurm_job_ids]
+    return ",".join(job_ids)
+
+
+def get_slurm_squeue_output(slurm_job_id_file: Path, use_ssh: bool = False) -> str:
```

Comment:
I'm a bit confused regarding the ssh option. What are we using in the automation? I'm not sure we need to support both of we only use one 🤔

Reply:
FE:
```
process = multiprocessing.Process(target=store.update_ongoing_analyses, kwargs={"use_ssh": True})
       
process = multiprocessing.Process(
            target=store.update_run_status, kwargs={"analysis_id": analysis_id, "use_ssh": True}
```

CLI:
```
context.obj["trailblazer"].update_run_status(analysis_id=analysis_id)
 and several other places
```

So both with and without depending who is calling.
Automation on Hasta do not use ssh